Sources/SwiftPhoenixClient/PhoenixTransport.swift

```diff
@@ -266,6 +266,13 @@ open class URLSessionTransport: NSObject, PhoenixTransport, URLSessionWebSocketD
     // The task has terminated. Inform the delegate that the transport has closed abnormally
     // if this was caused by an error.
     guard let err = error else { return }
+      
+    if let nsError = err as NSError? {
+        // If the client cancels a task, don't treat it as an abnormal error
+        if nsError.domain == "NSURLErrorDomain" && nsError.code == NSURLErrorCancelled {
+            return
+        }
+    }
```

Comment:
Could this use the `URLError` type like:
```suggestion
    if let urlError = err as? URLError, urlError.code == .cancelled {
        // If the client cancels a task, don't treat it as an abnormal error
        return
    }
```